tests/resources/Nimbus-Util.robot

```diff
@@ -138,6 +141,29 @@ Deploy Nimbus vCenter Server
     Close connection
     [Return]  ${user}-${name}  ${ip}
 
+Deploy Nimbus ESXi Server Async
+    [Tags]  secret
+    [Arguments]  ${user}  ${password}  ${name}  ${version}=${ESX_VERSION}
```

Comment:
I don't think we want the user and password here as an argument, as it could leak the user/pass in the calling keyword.  I think simplify and remove the args here and just pass %{NIMBUS_USER}/PASSWORD directly into secret sshpass command now.